anchors:
    - [0, 0]
    - [0.5, 0.4]
    - [0.34, 0]
transition_matrix: [1, 0, 0, 1]
observation_matrix: [1, 0, 0, 1]
initial_state: [0, 0]
initial_cov: [0.0001, 0, 0, 0.0001]
